src/objects/Mesh.js

```diff
@@ -49,6 +49,55 @@ class Mesh extends Object3D {
 
 	}
 
+	volume( precision ) {
+
+		let volume = 0;
+		const vertices = this.geometry.attributes.position.array;
```

Comment:
I don't think that your current code works because `fromBufferAttribute()` expects a buffer attribute, not an array. Rewrite this line as:
```js
const positionAttribute = this.geometry.attributes.position;
```